.github/workflows/test.yml

```diff
@@ -0,0 +1,45 @@
+name: Node CI
+
+on: [push, pull_request]
```

Comment:
I think this will run twice last I checked. You can be more specific and only run push on certain branches such as master.

Reply:
Indeed, it did run twice for this PR:

![image](https://user-images.githubusercontent.com/229881/66961302-5964e800-f03c-11e9-91cb-f6f655ecc3d9.png)

Reply:
Ok I'll copy it like in file-trace.